<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="3">
     <widget class="QGroupBox" name="groupBoxBotoes">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QPushButton" name="btnAbrirTeste">
         <property name="text">
          <string>Abrir Teste</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btnAbrirTreino">
         <property name="text">
          <string>Abrir Treinamento</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="btnTreinar">
         <property name="text">
          <string>Treinar</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="btnPredizer">
         <property name="text">
          <string>Predizer</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QGroupBox" name="groupBoxCondicaoParada">
      <property name="title">
       <string>Escolher Condição de Parada</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="1">
        <widget class="QLineEdit" name="inputErro"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="inputIteracoes"/>
       </item>
       <item row="0" column="0">
        <widget class="QRadioButton" name="btnErro">
         <property name="text">
          <string>Erro Máximo</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="btnNumIteracoes">
         <property name="text">
          <string>Número de Iterações</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBoxFuncao">
      <property name="title">
       <string>Escolher Função de Ativação</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QRadioButton" name="btnLogistica">
         <property name="text">
          <string>Logística</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="btnHiperbolica">
         <property name="text">
          <string>Hiperbólica</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="btnDefAuto">
         <property name="text">
          <string>Definir automaticamente</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="btnInserir">
         <property name="text">
          <string>Inserir</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Neurônios Oculta:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="inputNumOculta"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" colspan="5">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Saídas</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="1">
        <widget class="QTextBrowser" name="saidaTexto"/>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="labelImg">
         <property name="styleSheet">
          <string notr="true">border: 1px solid black;
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>920</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
